src/Services/Communities/Communify.Communities.ApiFunctions/CreateCommunityFunction.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading.Tasks;
+using Communify.Communities.Common;
+using Communify.Communities.Common.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+
+namespace Communify.Communities.ApiFunctions
+{
+    public class CreateCommunityFunction
+    {
+        private readonly ICommunityRepository _communityRepository;
+        public const string CreateFunctionName = "CreateCommunityFunction";
```

Comment:
Should this be a const here or should it be on the function? I don't think we'll end up re-using it will we?